src/main.cc

```diff
@@ -21,77 +22,76 @@ class Worker : public Nan::AsyncWorker {
     exit_code = WaitForChildProcessToExit(child_process, test_mode);
   }
 
-  void HandleOKCallback() {
-    Local<Value> argv[] = {Nan::New<Integer>(exit_code)};
-    callback->Call(1, argv, async_resource);
+  void OnOK() {
+    const std::vector<napi_value> argv{Napi::Number::New(Env(), exit_code)};
+    Callback().Call(argv);
   }
 };
 
-void GetAuthorizationForm(const Nan::FunctionCallbackInfo<Value>& info) {
+Napi::Value GetAuthorizationForm(const Napi::CallbackInfo& info) {
+  const Napi::Env& env = info.Env();
   auto auth_form = CreateAuthorizationForm();
-  auto buffer = Nan::CopyBuffer(auth_form.c_str(), auth_form.size());
-  info.GetReturnValue().Set(buffer.ToLocalChecked());
+  auto buffer = Napi::Buffer<char>::Copy(env, auth_form.c_str(), auth_form.size());
+  return buffer;
 }
 
-void ClearAuthorizationCache(const Nan::FunctionCallbackInfo<Value>& info) {
-  ClearAuthorizationCache();
+Napi::Value ClearAuthorizationCache(const Napi::CallbackInfo& info) {
+  ClearAuthorizationCacheImpl();
```

Comment:
I needed to rename this, otherwise the compiler would complain at https://github.com/atom/fs-admin/pull/105/files#diff-7b9337fcff3141736997afd31f7a2764b3538153922282c8407af5f43d543cabR90